### DISCUSSION TITLE
我想问下ipv6需要怎么使用，开启了yes

### DISCUSSION THREAD

**Discussion by Mxzjxs** [kkkgo/PaoPaoDNS#138] [问答和帮助]
我想实现 paopaodns识别纯ipv6的地址后，直接用国内方式访问，不用经过网关，这种能实现吗？我主路由开启ipv6后，paopaodns 的ipv6也设置为了yes，似乎返回的依然是v4地址，是我哪里没设置对吗

**Comment by kkkgo** [comment]
你的描述比较模糊，首先IPV6是有多个选项的，区别看文档，当设置为yes的时候仅对境内域名开放v6解析。
根据你模糊的描述我猜测可能你是想把返回fakeip的v6域名得到解析，那你可能需要设置only6，并且把对应域名加入force_dnscrypt_list.txt

**Comment by Mxzjxs** [comment]
> 你的描述比较模糊，首先IPV6是有多个选项的，区别看文档，当设置为yes的时候仅对境内域名开放v6解析。 根据你模糊的描述我猜测可能你是想把返回fakeip的v6域名得到解析，那你可能需要设置only6，并且把对应域名加入force_dnscrypt_list.txt

就是我要怎么设置才能使用ipv6呢。目前是主路由开启了v6，paopaodns也开启了yes，但是无法正常访问v6网站

**Reply by kkkgo** [reply]
能不能具体描述你的环境配置、docker配置环境变量、访问的哪个域名
